### PR TITLE
Added basicauth for admin

### LINKED ISSUE #943: Only activate the basicauth in the admin by default

### PR DESCRIPTION
**Pull request by VamsiSangam**
Fixes #943. Does my edit solve the issue?

### CODE REVIEW

File: kinto/plugins/admin/src/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as signoffPlugin from "kinto-admin/lib/plugins/signoff";
 
 const settings = {
   maxPerPage: 50,
-  singleServer: document.location.toString().split('/admin/')[0]
+  authMethods: ['basicauth']
```
Comment (leplatrem) [inline kinto/plugins/admin/src/index.js]: Both settings should be kept!

**Comment by VamsiSangam** [comment]
I am a noob. I made the necessary changes in my branch in my forked version ([VamsiSangam:edits](https://github.com/VamsiSangam/kinto/tree/edits)). So, do those commits automatically get included in this pull request to merge [VamsiSangam:edits](https://github.com/VamsiSangam/kinto/tree/edits) with [Kinto:master](https://github.com/Kinto/kinto)?

**Comment by Natim** [comment]
> do those commits automatically get included in this pull request to merge

Yes

**Comment by leplatrem** [comment]
Yes! exactly that's the way it works! it's pretty awesome once you figure out the workflow :)

We'll wait for the tests to pass, and you'll have your code integrated once merged :)

**Comment by VamsiSangam** [comment]
Oh yes, they did. Yeah @leplatrem, it is awesome 😄 ! Sure, thank you! 😄

**Pull request merged**